app/src/androidTest/java/com/monkeyteam/chimpagne/newtests/viewmodels/EventViewModelTests.kt

```diff
@@ -210,5 +213,7 @@ class EventViewModelTests {
 
     eventSearch2VM.deleteTheEvent(
         onSuccess = { assertTrue(true) }, onFailure = { assertTrue(false) })
+
+     */
```

Comment:
I don't this kind of comment block should be merged to main. Simple // TODO: are fine, 70 lines is too messy

Reply:
I do not understand your point. I commented the tests because they don't work 100% of the time due to the new database changes, since I worked on those test on a previous PR, so I will work on fixing them on a separate PR instead of doing it directly here.

Reply:
As @Athanaze said, this huge comment should not be here, remove it

Reply:
I removed it and replaced it with a comment to the PR where I fixed it.